module/zfs/spa.c

```diff
@@ -11100,6 +11221,14 @@ ZFS_MODULE_PARAM(zfs_livelist_condense, zfs_livelist_condense_, new_alloc, INT,
 	"Whether extra ALLOC blkptrs were added to a livelist entry while it "
 	"was being condensed");
 
+ZFS_MODULE_PARAM(zfs_spa, spa_, note_txg_time, UINT, ZMOD_RW,
+	"How frequently TXG timestamps are stored internally for scrubs "
```

Comment:
I'd not mention scrubs here.  I see other uses for this data.

Reply:
Makes sense!